cli/popper/commands/cmd_env.py

```diff
@@ -48,4 +55,13 @@ def cli(ctx, pipeline, add, rm):
         for e in rm.split(','):
             config['pipelines'][pipeline]['envs'].remove(e)
 
+    if ls:
+        web_results = urllib2.urlopen("https://hub.docker.com/v2/repositories/falsifiable/poppercheck/tags").read()
+        results = json.loads(web_results)['results']
+        environments = []
+        for result in results:
+            environments.append(result['name'])
+        list_of_environments = ", ".join(environments)
+        print list_of_environments
```

Comment:
there's a [utility function](https://github.com/systemslab/popper/blob/master/cli/popper/utils.py#L129) to pretty-print a YAML list, maybe we could use that by creating a YAML-formatted string and passing it to this function?